File: itcloud/src/main/java/com/itinfo/itcloud/controller/DashboardController.java
package com.itinfo.itcloud.controller;

import io.swagger.annotations.Api;
import lombok.RequiredArgsConstructor;
import lombok.extern.slf4j.Slf4j;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;

@Controller
@RequiredArgsConstructor
@Slf4j
@Api(tags = "dashboard")
public class DashboardController {

	@GetMapping("/dash")
	public String dashboard(){
		return "dashboard";
	}

	@GetMapping("/vnc")
	public String vnc(){
		return "vnc";
	}

}

File: itcloud/src/main/java/com/itinfo/itcloud/controller/DashboardController.kt
package com.itinfo.itcloud.controller;

import com.itinfo.common.LoggerDelegate

import io.swagger.annotations.Api
import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping

/**
 * [DashboardController]
 * 대시보드 관련
 */
@Controller
@Api(tags = ["dashboard"])
@RequestMapping("/api/v1")
class DashboardController: BaseController() {
	@GetMapping("/vnc")
	fun vnc(): String {
		log.debug("/vnc ...")
		return "vnc"
	}

	companion object {
		private val log by LoggerDelegate()
	}
}
